src/Cart.java

```diff
@@ -1,13 +1,22 @@
+import java.util.ArrayList;
+
 public class Cart {
     private Product product;
```

Comment:
If we add list of products, is there a need to include the single product here as well ?